lib/commons/color/get-background-color.js

```diff
@@ -121,6 +121,9 @@ function elmPartiallyObscured(elm, bgElm, bgColor) {
  * @param {Element} elm
  */
 function includeMissingElements(elmStack, elm) {
+	if (!elmStack.includes(elm)) {
+		elmStack.unshift(elm);
```

Comment:
Can you foresee any problems with putting the target at the front of the element stack? Do we need to check for z-index? This is shimming a failure by `document.elementsFromPoint` to pick up inline elements spanning line breaks.

Reply:
Since z-index could be manipulated by the page's styles the target element might actually be behind other elements in the stack. That seems problematic to me instinctually, but I am probably missing something.

Reply:
I'll add another test for that case and see if it causes problems.